test/utils.h

```diff
@@ -485,15 +485,26 @@ inline bool cudaArchGuardShouldSkip(
   }
 
 // util to track support matmul operand layout.
-using MatmulLayout = MmaOptions::MmaInputLayout;
+using MatmulLayout = MmaOptions::MmaLayout;
 
 static constexpr std::array<MatmulLayout, 3> kAllSupportedMatmulLayout = {
     MatmulLayout::TT,
     MatmulLayout::NT,
     MatmulLayout::TN};
 
+static constexpr std::array<MatmulLayout, 4> kAllSupportedMatmulLayoutAndNN = {
```

Comment:
This will eventually replace `kAllSupportedMatmulLayout` when NN support for Volta is done.